src/SequenceManager.cc

```diff
@@ -358,4 +392,55 @@ void SequenceManager::store_sequence(const Sequence& seq) const
         store_step(seq_path / extract_filename_step(++idx, max_digits, step), step);
 }
 
+std::string SequenceManager::stage_files_in_directory(std::filesystem::path dir_name, const std::string& filetype)
+{
+
+    // detect what has changed in the sequence
+    std::string git_msg{""};
+    auto stats = git_repo_.status();
+    for(const auto& elm: stats)
+    {
+        
+        // filter for changes in sequence
+        if (gul14::starts_with(elm.path_name, dir_name.string()))
+        {
+            if (elm.changes == "new file" and (filetype == elm.changes or filetype == ""))
+            {
+                git_msg += gul14::cat("\n", "- create '",elm.path_name, "'");
+            }
+            else if (elm.changes == "modified" and (filetype == elm.changes or filetype == ""))
+            {
+                git_msg += gul14::cat("\n", "- modify '",elm.path_name, "'");
```

Comment:
Just a minor thing:
```suggestion
                git_msg += gul14::cat("\n- modify '",elm.path_name, "'");
```
... saves a tiny bit of code at runtime. :)

Reply:
Addressed together with the missing blank before `elm.*`.